base/files/.welcome.sh

```diff
@@ -7,7 +7,6 @@ printf "Welcome to \033[1;38;5;208mStackStorm HA\033[0m \033[1m%s\033[0m (${UBUN
 printf " * Documentation: https://docs.stackstorm.com/\n"
 printf " * Community: https://stackstorm.com/community-signup\n"
 printf " * Forum: https://forum.stackstorm.com/\n"
```

Comment:
Enterprise string below kept extra new line separators. Let's use them.
```suggestion
printf " * Forum: https://forum.stackstorm.com/\n\n"
```